src/Workspaces/VisualBasic/Portable/Extensions/TypeSyntaxGeneratorVisitor.vb

```diff
@@ -129,10 +129,19 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Extensions
             Dim hasNames = Not names.IsDefault
 
             Return SyntaxFactory.TupleType(SyntaxFactory.SeparatedList(
-                types.Select(Function(t, i) SyntaxFactory.TupleElement(
-                    If(hasNames AndAlso names(i) IsNot Nothing, SyntaxFactory.IdentifierName(names(i)), Nothing),
-                    If(hasNames AndAlso names(i) IsNot Nothing, SyntaxFactory.Token(SyntaxKind.AsKeyword), Nothing),
-                    t.GenerateTypeSyntax()))))
+                types.Select(Function(t, i)
+                                 Dim element As TupleElementSyntax
```

Comment:
no need for this local.  just have the individual if blocks return the appropriate expression.